hack/builder/titus-executor-builder.sh

```diff
@@ -99,9 +98,9 @@ fpm -t deb -s dir -C root \
   --deb-field "Branch: ${git_sha}" \
   --deb-activate ldconfig \
   --depends libc6 \
-  --depends 'apparmor >= 2.12-4ubuntu7' \
+  --depends 'apparmor >= 2.12' \
   --depends 'docker-ce >= 5:18.09.1~3-0~ubuntu-xenial' \
-  --deb-recommends lxcfs \
+  --depends lxcfs \
```

Comment:
Do we want to make lxcfs a hard dep? IIRC, we can't (or don't?) use lxcfs while running in the test docker container on build kite.

Reply:
Hmm, ok - I can back this out and install it explicitly instead.